Files/ViewModels/AdaptiveSidebarViewModel.cs

```diff
@@ -1,13 +1,19 @@
-﻿using Files.UserControls;
+﻿using Files.Filesystem;
```

Comment:
Should this view model be renamed to SidebarViewModel?

Reply:
The view model contains most of the adaptive behavior for the sidebar which drove my naming decision. I don't think it's really that important as long as it contains the word sidebar in it.

Reply:
@duke7553 Back to the times where we had a hard time naming things 😆. I think it's best to call it SidebarViewModel because adding adaptive to the name can sound like it's limited to the adaptive behavior.